.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
+jdk:
+  - openjdk6
+  - openjdk7
```

Comment:
Both openjdk6 and openjdk7 seem to be failing all the time.

https://travis-ci.org/revans2/incubator-storm/jobs/59783142

This looks like a bug in openjdk and glibc combining together.  We may not want to test with them if it is not something that we cannot fix.